scripts/check_used_functions.py

```diff
@@ -34,14 +31,13 @@ def read_all_source():
 def check_functions(functions, source):
     for function in functions:
         if function not in source and function not in IGNORED:
-            print("Missing: " + function)
-            global ERROR
-            ERROR = True
+            error("Missing: " + function)
 
 
 if __name__ == '__main__':
-    functions = functions_list()
+    functions = all_functions()
     source = read_all_source()
     check_functions(functions, source)
-    if ERROR:
-        sys.exit(1)
+
+if ERROR:
```

Comment:
could you leave this in the `if __name__ == "__main__` part ?

Reply:
oops again